uk.co.compucorp.civicrm.hrleaveandabsences/views/manager-leave/components/manager-leave-requests.html

```diff
@@ -118,14 +118,19 @@
                     <i class="fa fa-ellipsis-v"></i>
                   </a>
                   <ul class="dropdown-menu dropdown-menu-right" uib-dropdown-menu>
-                    <li ng-repeat="action in ctrl.actionsFor(request)">
-                      <a ng-if="action === 'cancel'" href
+                    <li
+                      ng-repeat="action in ctrl.actionsFor(request)"
+                      ng-init="isDirectAction = ['cancel', 'approve', 'reject'].indexOf(action) !== -1">
```

Comment:
Angular doesn't like tildes unfortunately. https://stackoverflow.com/questions/40573781/the-tilde-operator-not-working-in-angularjs

```js
// that wouldn't work
~['cancel', 'approve', 'reject'].indexOf(action)
// that would work, but we compare numbers to booleans then, which is not good
['cancel', 'approve', 'reject'].indexOf(action)+1
// so we probably want to do this instead, but it is not well readable
!!(['cancel', 'approve', 'reject'].indexOf(action)+1)
// so I would suggest to compare the following expression to -1 instead
['cancel', 'approve', 'reject'].indexOf(action)
```

However, I don't think we want to compare numbers with booleans, so probably